openedx/core/djangoapps/api_admin/utils.py

```diff
@@ -1,5 +1,8 @@
 """ Course Discovery API Service. """
+from django.conf import settings
+
 from edx_rest_api_client.client import EdxRestApiClient
+from openedx.core.djangoapps.theming import helpers
```

Comment:
Yep, we are aware, and have some thoughts about how to refactor -- @saleem-latif, FYI -- it's also open source, so feel free to lend a hand!  :)